ios/CodePush/CodePush.h

```diff
@@ -1,6 +1,6 @@
-#import "RCTBridgeModule.h"
+#import <Foundation/Foundation.h>
```

Comment:
BTW, this change was necessary because we're using `NSObject` and were getting away with not importing this header because `RCTBridgeModule.h` imported it for us. Now that we've removed that dependency, we now need to explicitly import the `Foundation.h` file to ensure `NSObject` resolves correctly.